lib/runtime/wasmer/exports_test.go

```diff
@@ -119,6 +120,44 @@ func TestInstance_Version_NodeRuntime(t *testing.T) {
 	require.Equal(t, expected, version.RuntimeVersion)
 }
 
+func balanceKey(t *testing.T, pub []byte) []byte {
+	h0, err := common.Twox128Hash([]byte("System"))
+	require.NoError(t, err)
+	h1, err := common.Twox128Hash([]byte("Account"))
+	require.NoError(t, err)
+	h2, err := common.Blake2b128(pub)
+	require.NoError(t, err)
+	return append(append(append(h0, h1...), h2...), pub...)
+}
+
+func TestNodeRuntime_ValidateTransaction(t *testing.T) {
+	DefaultTestLogLvl = 4
+
+	alicePub := common.MustHexToBytes("0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d")
+	aliceBalanceKey := balanceKey(t, alicePub)
+
+	rt := NewTestInstance(t, runtime.NODE_RUNTIME)
```

Comment:
you can change this to load the gssmr genesis:
```
	gen, err := genesis.NewGenesisFromJSONRaw("../../../chain/gssmr/genesis-raw.json")
	require.NoError(t, err)

	genTrie, err := genesis.NewTrieFromGenesis(gen)
	require.NoError(t, err)

	// set state to genesis state
	genState := storage.NewTestTrieState(t, genTrie)

	cfg := &Config{}
	cfg.Storage = genState
	cfg.LogLvl = 5

	instance, err := NewRuntimeFromGenesis(gen, cfg)
	require.NoError(t, err)
```

Reply:
The test is running without the genesis. The balance data was incorrect. I updated it to `0x000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000`